locale/en.json

```diff
@@ -165,6 +165,8 @@
   "IDENTIFICATION_FAILED": "Image identification failed.",
   "SIGHTING_START": "Sighting start",
   "SIGHTING_END": "Sighting end",
+  "NOT_READY_FOR_COMMIT": "This sighting is pending",
+  "NOT_READY_FOR_COMMIT_DESCRIPTION": "Sightings must remain in 'pending' state until detection is complete and all annotations are assigned.",
```

Comment:
assigned to what? Animals? Individuals?

Reply:
to animal clusters // encounters. you think it's confusing as-is?

Reply:
I think it should be clearer, yes.